### PR TITLE
Fix index field count

### PR DESCRIPTION
**Pull request by hartfordfive**
This change applies a small modification that adds 1 to the field count prior to recursively counting the other nested fields in the mapping. (Issue #674)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+
+## 1.5.1 / 2022-07-29
```
Comment (sysadmind) [inline CHANGELOG.md:2]: Version gets changed when we are ready for a release. 
```suggestion
## master / unreleased
```

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-1.5.0
```
Comment (sysadmind) [inline VERSION:1]: This should not be changed.

**Review by sysadmind — CHANGES_REQUESTED** [review]
You're missing the DCO. The check has instructions for how to fix. 
I'm not sure this takes into account a situation where both type and properties are set. I'm not sure if these are mutually exclusive, but if not these would potentially double count those fields.

**Comment by hartfordfive** [comment]
@sysadmind , thank you for the quick feedback.   I will address those required changes you pointed out.   

As for counting a mapping where both type and property are set, although the type `object` could be technically be present with the `properties` field, it isn't usually defined explicitly (at least based on my experience) as that's the default value as per the [documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/object.html).
```
You are not required to set the field type to object explicitly, as this is the default value.
```
Having said that, the possibility is a valid edge case that should be accounted for, so I'll go ahead and make the modification.  Thank you for bringing that up.

**Review by sysadmind — APPROVED** [review]
LGTM Thanks!

**Pull request merged**